woocommerce/woocommerce
# Remove variations logic from the AttributeControl component

### All Submissions:






### Changes proposed in this Pull Request:

* Removes any logic around variations from the `AttributeControl`
* Renames state and components to not use verbs unless they are actions
* Adds callback props around addition/removal/etc of attributes

Closes #36515  .






### How to test the changes in this Pull Request:



1. Navigate to the new product management experience
2. Enable tracks output in the console: `localStorage.setItem( 'debug', 'wc-admin:*' );`
3. Smoke test the attributes under the General tab and options under the Options tab
4. Make sure each works as expected and has the expected text ("options" vs "attributes") for each
5. Make sure the existing tracks events fire correctly for attributes and options respectively



### Other information:




### FOR PR REVIEWER ONLY:
